client/src/app/site/users/components/user-import/user-import-list.component.html

```diff
@@ -199,6 +199,20 @@ <h3>{{ 'Preview' | translate }}</h3>
                     </mat-icon>
                 </div>
             </div>
+
+            <!-- handle groups -->
+            <div *pblNgridCellDef="'newEntry.csvGroups'; value as groups">
+                <span *ngFor="let group of groups; let i = index">
```

Comment:
You can use `...;let last = last">` and than `<span *ngIf="!last">
(https://angular.io/api/common/NgForOf#local-variables)